controllers/userStatus.js

```diff
@@ -65,6 +76,9 @@ const getUserStatus = async (req, res) => {
 const getAllUserStatus = async (req, res) => {
   try {
     const { allUserStatus } = await userStatusModel.getAllUserStatus(req.query);
+    if (req.query.sorted) {
```

Comment:
I could not get this, why are we checking for query.sorted ?

Reply:
because when the user passes `sorted=true` in the URL, this function will run